.travis.yml

```diff
@@ -1,25 +1,30 @@
 language: elixir
 elixir:
-  - 1.6.6
   - 1.7.2
+  - 1.8.2
+  - 1.9.0
 
 otp_release:
   - 20.3
-  - 21.0
+  - 22.0
 
 script:
   - bin/fetch-configlet
   - bin/configlet lint .
   - mix deps.get
   - mix test
 
-  # fail if there are any compiler warnings
-  - mkdir ./tmp
-  - EXERCISM_TEST_EXAMPLES=true elixirc --warnings-as-errors **/example.exs **/*_test.exs -o ./tmp > /dev/null
-  - rm -rf ./tmp
+  # OLD - fail if there are any compiler warnings
+  # - mkdir ./tmp
+  # - EXERCISM_TEST_EXAMPLES=true elixirc --warnings-as-errors **/example.exs **/*_test.exs -o ./tmp > /dev/null
+  # - rm -rf ./tmp
+
+  # NEW - fail if there are any compiler and test problems for individual tests.
+  - bin/test_exercises.sh
```

Comment:
Since we have this in git history anyway, I would prefer to simply delete the commented out code and not label line 22 with `NEW - `

Reply:
For sure, I had left it commented to convert the change as I have in my personal practice, but I forget how rich GitHub is for looking at these things. I will amend.

Reply:
Re line 15, good point. I think we can get rid of it. Had left it because it worked and was fast, but if it were to fail Im not sure it would be helpful output.